Automattic/jetpack
# Writing prompts: remove the placeholder prompt and setting

## Proposed changes:

- Removes the placeholder blogging prompt and setting, following up on user feedback that this was confusing and/or distracting
- Adds an additional tag for posts that respond to writing prompts: `#dailyprompt-{prompt_id}`


### Other information:


## Jetpack product discussion

pe7F0s-uo-p2

## Does this pull request change what data or activity we track or use?

No.

## Testing instructions:

- Go to Settings > Writing, and you should no longer see the checkbox for displaying a placeholder prompt
- Open a new post in the editor--you should not see a placeholder writing prompt
- Add the `?answer_prompt=1830` query param to the `post-new.php` screen--you should still see a writing prompt in a pullquote block, for users who respond to a My Home writing prompt card or notification
- When adding the query param, you should see the tags `#dailyprompt` and `#dailyprompt-{prompt_id}` added to the post
